### PR TITLE
Add 1fichier and Journal du geek and remove Vine

### PR DESCRIPTION
**Pull request by alrjunior**
Adding 1fichier (hard) and Journal du Geek (easy) and removing Vive, finished in 2019 - https://en.wikipedia.org/wiki/Vine_(service)

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -6893,6 +6903,17 @@
         ]
     },
 
+    {
+        "name": "Journal du geek",
+        "url": "http://www.journaldugeek.com/editer-profil/",
+        "difficulty": "easy",
+        "notes": "Log in. Go to your profil, scroll down and click 'Supprimer mon profil'",
```
Comment (tupaschoal) [inline _data/sites.json:6910]:
```suggestion
        "notes": "Log in. Go to your profile, scroll down and click 'Supprimer mon profil'",
```

File: _data/sites.json [hunk 2]
```diff
@@ -6893,6 +6903,17 @@
...
+        "notes_fr": "Loggez vous, cliquez sur 'Modifiez mon profil', descendez en bas et cliquez sur 'Supprimer mon profil'",
```
Comment (tupaschoal) [inline _data/sites.json:6911]: Just to confirm, are you fluent in French? We ask people to only translate languages they can speak

Comment (alrjunior) [reply _data/sites.json:6911]: Fluent, I'm not. But I can read something in French.

Comment (tupaschoal) [reply _data/sites.json:6911]: Ok

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**